### ISSUE TITLE
Hyperparameters for single & multitask training

### ISSUE THREAD

**Issue by rakhimovv** [zhouxian/act3d-chained-diffuser#10]
Hello! Thank you for the code and the interesting results.

I have attempted to reproduce the results and noticed a lack of hyperparameters for both single and multitask training. I did not find any information on the difference between single and multitasks setup in either the code or the paper.

Could you please clarify the difference? Are the hyperparameters the same for both settings?

**Comment by QazyBi** [comment]
You need to pass the list of tasks separated by a space in a string like this: `python main_keypose.py --task="open_drawer push_button ..." ...`

**Comment by swords123** [comment]
Are other hyper-parameters consistent during multitasking training? Such as the learning rate?

**Comment by nickgkan** [comment]
Hi,

You can now find updated train/test scripts and checkpoints in our codebase for our new project here https://github.com/nickgkan/3d_diffuser_actor. We include the checkpoint for Act3D as well. You can find all the hyper-parameters for multi-task training there. For single-task training, you can use the same hyperparameters, both for Act3D and the new 3D Diffuser Actor.

That said, our new model outperforms Act3D by >16%, so check it out if you're interested! Please redirect your questions about Act3D and/or our new work on that repo.

Thank you for your interest in our work!

**Comment by rakhimovv** [comment]
@nickgkan Awesome, thank you!
